Source/Core/Core/BootManager.cpp

```diff
@@ -378,9 +378,9 @@ bool BootCore(const std::string& _rFilename)
   // Apply overrides
   // Some NTSC GameCube games such as Baten Kaitos react strangely to
   // language settings that would be invalid on an NTSC system
-  if (!StartUp.bOverrideGCLanguage && ntsc)
```

Comment:
Since the `ntsc` variable only is used in one place now instead of two, you can replace the only usage of it with `DiscIO::IsNTSC(StartUp.m_region)` and remove the definition.